crates/db_schema/src/impls/community.rs

```diff
@@ -194,6 +193,20 @@ impl Community {
       .await?;
     Ok(())
   }
+
+  pub async fn get_random_local_community(pool: &mut DbPool<'_>) -> Result<Option<Self>, Error> {
+    let conn = &mut get_conn(pool).await?;
+    sql_function!(fn random() -> Text);
+    community::table
+      .filter(community::local)
+      .filter(not(community::deleted))
+      .filter(not(community::removed))
+      .order(random())
+      .limit(1)
+      .first::<Self>(conn)
+      .await
+      .optional()
```

Comment:
Get rid of optional (see https://github.com/LemmyNet/lemmy/pull/5031)

Reply:
Done.